modules/appeals_api/app/controllers/concerns/appeals_api/character_validation.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+module AppealsApi
+  module CharacterValidation
+    extend ActiveSupport::Concern
+
```

Comment:
Do you think it would be worth it to write a separate standalone spec for this concern?

Reply:
oh, yes. oy. i had it and then reverted and didn't replace it. thank you for catching this!!